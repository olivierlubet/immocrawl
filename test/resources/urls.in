http://www.seloger.com/list.htm?idtt=2&naturebien=1,2,4&idtypebien=1,2&ci=920049,940003,940037&prox=1&tri=initial
#test de commentaire

http://www.seloger.com/list.htm?idtt=2&naturebien=1,2,4&idtypebien=1,2&ci=450234&prox=1&tri=initial
http://www.seloger.com/list.htm?idtt=2&naturebien=1,2,4&idtypebien=1,2&ci=370261&prox=1&tri=initial