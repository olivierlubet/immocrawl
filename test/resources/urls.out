http://www.seloger.com/list.htm?idtt=2&naturebien=1,2,4&idtypebien=1,2&ci=450234&prox=1&tri=initial
http://www.seloger.com/list.htm?idtt=2&naturebien=1,2,4&idtypebien=1,2&ci=370261&prox=1&tri=initial
https://www.google.fr
